spring-kafka/src/main/java/org/springframework/kafka/listener/DeadLetterPublishingRecoverer.java

```diff
@@ -75,7 +76,7 @@ public class DeadLetterPublishingRecoverer extends ExceptionClassifier implement
 	protected final LogAccessor logger = new LogAccessor(LogFactory.getLog(getClass())); // NOSONAR
 
 	private static final BiFunction<ConsumerRecord<?, ?>, Exception, TopicPartition>
-		DEFAULT_DESTINATION_RESOLVER = (cr, e) -> new TopicPartition(cr.topic() + ".DLT", cr.partition());
+		DEFAULT_DESTINATION_RESOLVER = (cr, e) -> new TopicPartition(cr.topic() + "-dlt", cr.partition());
```

Comment:
`-DLT` looks better. That way, it is almost identical to the old naming convention, except for the new delimiter.

Reply:
This means that all the tests in this PR need to be modified.

Reply:
@sobychacko  I think if we use "**-DLT**", it might cause compatibility issues with the retry topics that currently use "**dlt**" in lowercase. Don't you think so? Also, I don't see much need to use uppercase here.

Reply:
Thats OK then. Let's keep it as it is.